otherlibs/unix/unix.mli

```diff
@@ -879,7 +879,7 @@ val open_process_args_in : string -> string array -> in_channel
    command to run, and the second argument specifies the argument array passed
    to the command.  This function runs the command in parallel with the program.
    The standard output of the command is redirected to a pipe, which can be read
-   via the returned input channel.
+   via the returned input channel. The program is looked up in the [PATH].
```

Comment:
I think we would need a @since marker for the PATH lookup, or at least a clear indication in the documentation that older versions of OCaml may not do it.

Reply:
I added a clarification about this to the docstring, but without using a `@since` marker since it wasn't clear to me how to specify more than one such marker.